services/tracing/tracing/application_test.go

```diff
@@ -0,0 +1,170 @@
+package tracing
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"sensorbucket.nl/sensorbucket/pkg/pipeline"
+)
+
+func TestTracingSuite(t *testing.T) {
+	suite.Run(t, new(tracingSuite))
+}
```

Comment:
I recommend only using Suite's if an explicit Setup and Teardown function for all tests are required (for example testcontainers). 

The downside of TestSuite is that testing extensions cannot execute them independently.